.github/workflows/publish-release-artifact.yml

```
@@ -0,0 +1,205 @@
+name: Publish artifact
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  determine-version:
+    runs-on: ubuntu-latest
+    outputs:
+      project-version: ${{ steps.set-version.outputs.project-version }}
+      is-lts: ${{ steps.set-version.outputs.is-lts }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Set version
+        id: set-version
+        run: |
+          version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+
+          dot_count=$(echo $version | sed 's/[^.]//g' | awk '{ print length }')
+          if [ "${dot_count}" = '2' ]
+          then
+            is_lts=true
+          else
+            is_lts=false
+          fi
+
+          echo "Version is $version, is_lts: $is_lts"
+
+          echo "::set-output name=is-lts::${is_lts}"
+          echo "::set-output name=project-version::$version"
+  war:
+    runs-on: ubuntu-latest
+    needs: determine-version
+    steps:
+      - name: Fetch war
+        id: fetch-war
+        run: |
+          FILE_NAME=jenkins.war
+          PROJECT_VERSION=${{ needs.determine-version.outputs.project-version }}
+          IS_LTS=${{ needs.determine-version.outputs.is-lts }}
+          echo $PROJECT_VERSION
+          echo "::set-output name=file-name::$FILE_NAME"
+
+          REPO=war
+          if [ ${IS_LTS} = 'true' ]
+          then
+            REPO=war-stable
+          fi
+
+          echo $REPO
+
+          wget -q https://get.jenkins.io/${REPO}/${PROJECT_VERSION}/jenkins.war
```

```suggestion
          wget -O $FILE_NAME -q https://get.jenkins.io/${REPO}/${PROJECT_VERSION}/jenkins.war
```

for robustness/clarity?

---

wget will always use the file name, not sure it makes much different

---

Alternately,
```suggestion
          wget -q https://get.jenkins.io/${REPO}/${PROJECT_VERSION}/${FILE_NAME}
```